search/query_language_whitebox_test.go

```diff
@@ -0,0 +1,236 @@
+package search
+
+import (
+	"encoding/json"
+	"runtime/debug"
+	"testing"
+
+	c "github.com/fabric8-services/fabric8-wit/criteria"
```

Comment:
> and workitem as well I suppose...

No I first thought that the `And` and `Equals` function where sitting under the `workitem` package. I've imported the `workitem` package as usual in 5c7cfea1 now.